### ISSUE TITLE
Carthage failed to build

### ISSUE THREAD

**Issue by ghost** [sonsongithub/reddift#222]
Carthage failed to build. Seem to be missing "test_config.json"

** BUILD FAILED **

The following build commands failed:
	CpResource /.../Carthage/Checkouts/reddift/test/test_config.json /.../Library/Caches/org.carthage.CarthageKit/DerivedData/10.1_10B61/reddift/2.0.3/Build/Products/Release-iphonesimulator/reddift-iOSTests.xctest/test_config.json

**Comment by sonsongithub** [comment]
Test needs test_config.json file.
You can make it by editing `./test/test_config.json.sample`.

**Comment by ghost** [comment]
Yes, it works after I include test_config.json. But it does not automatically build for others if you include reddift in project. Just FYI.
